blocks.md

```diff
@@ -0,0 +1,116 @@
+
+A Block Plugin must contain a block, and a minimum of other supporting code. It may not contain any UX outside of the editor, such as WordPress options or `wp-admin` menus. Server-side code should be kept to a minimum.
+
+Plugins that only extend or provide styles for other, pre-existing blocks are currently not eligible for inclusion in the Block Directory. At this time, they are not supported by the Block Editor's seamless install process. Only Block Plugins that register a new block are currently supported.
+
+<h4>2. Block Plugins are separate blocks.</h4>
+
+Block plugins are intended to be single purpose, separate, independent blocks, not bundles or compendiums of many blocks. In most cases, a Block Plugin should contain only one single top-level block. The Block Directory will not include collections of blocks that could be reasonably split up into separate, independent, blocks.
+
+Block plugins may contain more than one block where a clearly necessary parent/child or container/content dependency exists; for example a list block that contains list item blocks.
+
+<h4>3. Plugin and block names should reflect the block's purpose.</h4>
+
+Plugin titles and block titles should describe what the block does in a way that helps users easily understand its purpose. In most cases the plugin title and the block title should be identical or very similar. Some examples of good plugin and block names include:
+
+`Rainbow Block`
+`Sepia Image Grid`
+`Business Hours Block`
+
+Please avoid plugin and block titles unrelated to the block itself, or that cannot be easily distinguished from core blocks. Some examples of unhelpful plugin and block names are things such as:
+
+`PluginCo Inc Block`
+`Widget`
+`Image Block`
+
+The same trademark restrictions for plugin titles exist for block titles and names. This means that a block may not be named "Facerange Block" unless developed by a legal representative of Facerange.
+
+<h4>3a. Block names should be unique and properly namespaced.</h4>
+
+The block name (meaning the [`name` parameter to `registerBlockType()`](https://developer.wordpress.org/block-editor/developers/block-api/block-registration/#block-name) and [`name` in `block.json`](https://github.com/WordPress/gutenberg/blob/master/docs/rfc/block-registration.md#name)) must be unique to the block. As with titles, please respect trademarks and other projects' commonly used names, so as not to conflict with them.
+
+The namespace prefix to the block name should reflect either the plugin author, or the plugin slug. For example:
+
+`name: "my-rainbow-block-plugin/rainbow"`, or
+`name: "john-doe/rainbow"`, or
+`name: "pluginco/rainbow"`.
+
+The namespace may not be a reserved one such as `core` or `wordpress`.
+
+<h4>4. Block Plugins must include a <code>block.json</code> file.</h4>
+
+The Block Registration RFC outlines the format of a `block.json` file: https://github.com/WordPress/gutenberg/blob/master/docs/rfc/block-registration.md
```

Comment:
It appears that this link doesn't exist anymore.

Reply:
One alternative might be the handbook page for the `registerBlockType` API:
https://developer.wordpress.org/block-editor/developers/block-api/block-registration/

But there isn't anything on that page that draws a direct connection to the block.json file.

Reply:
Looks like it was actually moved to this file: https://github.com/WordPress/gutenberg/blob/master/docs/designers-developers/developers/block-api/block-metadata.md